### PR TITLE
Initial page and link on the README to the procurement terms page.

### PR DESCRIPTION
**Pull request by robbiethegeek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,5 +13,6 @@ The following guides outline how we operate.
 * __Growth__
   * [Opportunity Tracking](growth/opportunities.md) - how we track business development opportunities
   * [Proposals](growth/proposals.md) - how we work together, internally or with team members, to produce a proposal
+  * [Terms](growth/terms.md) - the nomanclature, verbiage and acronyms associated with procurement
```
Comment (bengm) [inline README.md:16]: nomenclature

File: growth/terms.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Procurement Terms
+
+| Term / Acronym | Definition / Explaination |
+|----|---|
+| ATO | Authority to Operate |
+| Prime | In a contracting context the lead company is often known as the Prime or Primary contacting company.|
+| Sub | In a contracting context companies that are supporting a prime company / contractor are known subs.|
```
Comment (bengm) [inline growth/terms.md:7]: add at end `short for 'subcontractor'`

File: growth/terms.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+| Small | A designation based on the amount of income the business has made over time. There is a specific amount of revenue over time that determines if you are a small business. We are a small business. | 
+| Large | A designation based on the amount of income the business has made over time. There is a specific amount of revenue over time that determines if you are a large business. | 
+|NAICS code | A NAICS (pronounced NAKES) Code is a classification within the North American Industry Classification System. The NAICS System was developed for use by Federal Statistical Agencies for the collection, analysis and publication of statistical data related to the US Economy.| 
+|Set Aside| When a specific contract is "set aside" for a specific socio-economic designation, meaning that not all businesses can bid on the project only those businesses that meet the criteria for a specific designation. |
+| WOSB | Women owned small business.  We are a WOSB but don't have not been certified by the Small Business Administration (SBA) | 
+| 8a | A socio-economic designgation from SBA to level the playing field for socially and economically disadvantaged people or entities.|
```
Comment (bengm) [inline growth/terms.md:13]: 8(a)

**Review by bengm — CHANGES_REQUESTED** [review]
Made a few minor suggestions and corrections

**Comment by robbiethegeek** [comment]
@bengm all updates made.

**Review by bengm — APPROVED** [review]

**Pull request merged**